### PR TITLE
[TWAP] Update Hermes binary response format, add TwapMessage parsing to price-service-sdk

### PR DESCRIPTION
**Pull request by tejasbadadare**
- Update the Hermes binary output for the `/twap/:window_seconds/latest` endpoint for better efficiency and usability. 
  - New format:
  ```
  binary.data = [start_updatedata, end_updatedata]
  ```
  - Each updatedata contains a VAA and the merkle proofs for the requested price feeds. This lets us transmit just two VAAs total - one for the start block and one for the end block, regardless of how many price feeds were queried. In the previous format, the VAAs were being duplicated per requested price feed.

- Add TwapMessage parsing support to the Price Service SDK. This will be used by the Solana Receiver SDK when handling binary TwapMessages from Hermes, similar to how PriceFeedMessages are currently being parsed and used.

### CODE REVIEW

File: price_service/sdk/js/src/AccumulatorUpdateData.ts [hunk 1]
```diff
@@ -64,6 +76,41 @@ export function parsePriceFeedMessage(message: Buffer): PriceFeedMessage {
   };
 }
 
+export function parseTwapMessage(message: Buffer): TwapMessage {
```
Comment (keyvankhademi) [inline price_service/sdk/js/src/AccumulatorUpdateData.ts:79]: let's add a simple test for this.

File: price_service/sdk/js/src/__tests__/AccumulatorUpdateData.test.ts [hunk 2]
```diff
@@ -113,4 +114,38 @@ describe("Test parse accumulator update", () => {
...
+    );
+
+    // Test that both messages are parsed successfully
+    const twapMessage1 = parseTwapMessage(updates[0].message);
+    expect(twapMessage1.feedId.toString("hex")).toBe(
+      "49f6b65cb1de6b10eaf75e7c03ca029c306d0357e91b5311b175084a5ad55688"
+    );
+    expect(twapMessage1.cumulativePrice.toString()).toBe("1760238576144013");
+    expect(twapMessage1.cumulativeConf.toString()).toBe("5113466755162");
+    expect(twapMessage1.numDownSlots.toString()).toBe("72037403");
```
Comment (guibescos) [inline price_service/sdk/js/src/__tests__/AccumulatorUpdateData.test.ts:133]: big number here since stocks stop trading every night

**Review by guibescos — APPROVED** [review]

File: price_service/sdk/js/src/__tests__/AccumulatorUpdateData.test.ts [hunk 3]
```diff
@@ -113,4 +114,38 @@ describe("Test parse accumulator update", () => {
       "Invalid accumulator message"
     );
   });
+
+  test("Parse TWAP message", () => {
+    // Sample data from the Hermes latest TWAP endpoint.
+    const test_accumulator_data_twap =
```
[the remark this answers is not in the document]
Comment (tejasbadadare) [reply price_service/sdk/js/src/__tests__/AccumulatorUpdateData.test.ts:120]: woops, too much python & rust

**Review by ali-bahjati — APPROVED** [review]
LGTM!

**Pull request merged**